### ISSUE TITLE
Minor documentation issue

### ISSUE THREAD

**Issue by demolitions** [SimulaVR/Simula#172]
Really just a minor issue regarding installation time.
The README suggests a "less than 1 minute" installation, while:

on a fresh Arch Linux install (installed on a spare SSD just for the purpose of trying Simula),
with `nix` installed in the official way, 
`cachix` installed in the official way, 

launching the installation process initially "hangs" without any feedback (it's doing something, `nix-build` is using around 50% CPU on 4 cores) for around 8 minutes at the message:

```
[user@hostname Simula]$ source ./utils/Helpers.sh && installSimula
/home/user/.nix-profile/bin/nix
nix already installed..
/home/user/.nix-profile/bin/cachix
cachix already installed..
Configured https://simula.cachix.org binary cache in /home/user/.config/nix/nix.conf

Message from Simula Contributors.
=================================

In order to show project funders that Simula is useful, 
we anonymously collect the following data:

- Session length
- Number of windows active
- Error logs on crashes

```
This on a Ryzen 7 2700 with 32GB of RAM, then it finally goes on with saying:
```
these 9 derivations will be built:
  /nix/store/9qksxy3iglvq1viqaqpr0a4pmd1n2g8f-xwayland-21.1.1.drv
  /nix/store/cmykn1y1i9bg7aqkd0mnbr48yrg9fn66-nixGLIntel.drv
  /nix/store/xv0q6wa32m2nrxbsx8ln9pxc6akhqagf-xvfb-run.drv
  /nix/store/2cnvlmbpk2dyvnk6jd78zv6c5mcl41iq-godot-3.2.drv
  /nix/store/7gr9xyg36fh8smvhq8gjn2h73kfbzi5f-pernosco_submit.drv
  /nix/store/ri3l2wq7azqqi1z4k5d04cqdp2q808yg-midori.drv
  /nix/store/vcpxqinlf913lhjpza5s8p200nwnncqv-i3status.drv
  /nix/store/zirzk4h1z3z5fwh8ys8ajyw3cxhmdcji-xfce4-terminal.drv
  /nix/store/hmcwa7ggi1qalf87j760q8bpdmykcmvq-Simula.drv

```
(Funny thing - I use xfce4 as my DE, and it's actually running the compilation inside an xfce4-terminal window, yet it wants to compile it again)

Then it downloads 1.3 GB of packages from the cache (thanks cachix).

I'm now in the middle of the compilation of those 9 packages, 25 minutes in from the `installSimula` command, fortunately it seems that it's using full multiprocessing, so I hope it won't take too long, even if godot is pretty big to compile.

Just downloading 1.3GB of packages took almost 5 minutes, and I have a standard fiber connection.

I just feel that saying "less than 1 minute" is at least misleading, I'd change that to a more realistic time estimation, it's not a nice presentation.

**Comment by demolitions** [comment]
Update: it didn't take long, in 31 minutes I have the whole lot installed, now to try and run it.

**Comment by georgewsinger** [comment]
Sorry about this.  This means cachix is failing to fetch the binaries, and the installation is reverting to a normal nix build (which yeah, can take up to an hour or more depending upon how fast your machine is).

Sometimes this is our fault.  Sometimes cachix sporadically fails and then starts working again a few hours later.  :S 

Are you on `master` branch?

**Comment by demolitions** [comment]
Yes, I am on master, however, cachix has downloaded most of the packages, around 1.3GB, and only that takes a bit more than 1 minute on a 200Mbps connection.

I'd explore better (or maybe just give the user a heads up in the installer) the 8-ish minute pause before starting, since it actually appears that the installer just freezes, if you are not a paranoid maniac like me and have a performance monitor on your screen at all times.

It all works perfectly, however, in the end the software is installed and running perfectly, it's only a really minor documentation issue, I think that just addressing it by echo-ing some messages in the installer will be enough.
